website/docs/r/global_cluster_config.html.markdown

```diff
@@ -13,6 +13,7 @@ description: |-
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
 
+~> **IMPORTANT:** A Global Cluster Configuration, once created, can only be deleted and never re-created again with the same data if not without the help of the Atlas UI. This is because the configuration and its related collection with shard key and indexes are managed separately and they would end up in an inconsistent state. [Read more about Global Cluster Configuration](https://www.mongodb.com/docs/atlas/global-clusters/)
```

Comment:
i guess we can't prevent the resource for being created again once deleted?  it'd be nice if we could detect that conditions and block the create (the second time)

Reply:
yes, but I can't think of a way on our side. It should be something the Atlas itself need to implement a block at API level.